index.html

```diff
@@ -40,107 +49,128 @@ <h4 class="headline">LET’S CONNECT</h4>
   <div class="cards-container">
     <ul class="cards-list flex-center-culomn">
       <li class="card-item flex-center-culomn">
-        <img class="card-image" src="img/Snapshoot-Portfolio1.png" alt="Portfolio Portfolio">
-        <div class="list-conatiner">
-          <h2>Tonic</h2>
-           <div class="list">
-            <span class="list-item item1">Canopy</span>
-            <span class="list-item">back end Dev</span>
-            <span class="list-item">2015</span>
-           </div>
-        </div>
-        <div class="text-container">
-          <p>
-            A daily selection of privately personalized reads; no accounts or sign-ups required.
-          </p>
+        <div class="img-container">
```

Comment:
- [ ] Kindly use `grid` to display your projects. As described in the requirement You should use `flex` and `grid`